WalletWasabi/Blockchain/Transactions/TransactionStore.cs

```diff
@@ -36,7 +38,7 @@ public TransactionStore(string workFolderPath, Network network, bool migrateData
 		{
 			string oldPath = Path.Combine(workFolderPath, "Transactions.dat");
 			Logger.LogInfo($"Migration of transaction file '{oldPath}' to SQLite format is about to begin. Please wait a moment.");
-			Import(oldPath, network, deleteAfterImport: false);
+			Import(oldPath, dbPath, network, deleteAfterImport: true);
```

Comment:
Delete the file after import so the problem doesn't occur anymore. Probably this parameter should be removed altogether, and we always delete after import